packages/httpsnippet-client-api/src/index.ts

```diff
@@ -143,10 +145,16 @@ const client: Client<APIOptions> = {
     }
 
     let sdkPackageName;
-    let sdkVariable;
+    let sdkVariable: string;
     if (opts.identifier) {
       sdkPackageName = opts.identifier;
-      sdkVariable = opts.identifier;
+
+      sdkVariable = camelCase(opts.identifier);
+      if (isReservedOrBuiltinsLC(sdkVariable)) {
+        // If this identifier is a reserved JS word then we should prefix it with an underscore so
+        // this snippet can be valid code.
+        sdkVariable = `_${sdkVariable}`;
```

Comment:
***great*** call here

Reply:
yeah we do something similar for method and type names in codegen